backend/qemu.pm

```diff
@@ -186,8 +187,10 @@ sub open_file_and_send_fd_to_qemu {
     my ($self, $path, $fdname) = @_;
     my $rsp;
 
+    mkpath(dirname($path));
     my $fd = POSIX::open($path, POSIX::O_CREAT() | POSIX::O_RDWR());
-    die "Failed to open $path: $!" unless (defined $fd);
+    use Carp;
```

Comment:
I'd rather add the `use` at the top of the file. It will be evaluated at compile time anyways.

Reply:
oops. That was only for temporary debugging. Removed the change.